### DISCUSSION TITLE
What news about  PH7CMS

### DISCUSSION THREAD

**Discussion by Salva99999** [pH7Software/pH7-Social-Dating-CMS#1168] [General]
Hi everyone, I've installed version 18 and customised it. I haven't done all the translations yet. I have subscribed to a monthly payment on "patreon" but the project has stopped? I don't see much movement in the posts. I hope I've made the right choice

**Comment by SosthenesAlpha** [comment]
> Hi everyone, I've installed version 18 and customised it. I haven't done all the translations yet. I have subscribed to a monthly payment on "patreon" but the project has stopped? I don't see much movement in the posts. I hope I've made the right choice

Hello and thanks for this feedback. I just want to ask, how did you manage to get it to work on your webserver ? For me, it's NOT working on webserver. It works on well localhost but that is all there is to it. The particular problem I have is with the registration page. It doesn't load options to select gender, age and other info. It simply directs user to login page and forces woman gender and age on them. 🤷🏿‍♂️
I have read documentations to stupor, yet nothing works. I have not seen people talk about this here.
